styles.css

```diff
@@ -0,0 +1,111 @@
+body {
+  font-family: "Bebas Neue", sans-serif;
+  margin: auto;
+  font-size: 1.3rem;
+}
+
+h1 {
+  text-align: center;
+  font-size: 3rem;
+  background-color: #b1c5c3;
+  margin-top: 0;
+  padding: 10px;
+}
+
+.questionBox {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  margin-bottom: 30px;
+}
+
+.img-bookmark-white {
+  height: 40px;
+  position: absolute;
+  top: -18px;
+  right: 45px;
+}
+
+.questionCard {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  text-align: center;
+  flex-wrap: wrap;
+  align-items: left;
+  width: 70vw;
+  height: 25vh;
+  border: 1px solid black;
+  border-radius: 10px;
+  margin: auto;
+  padding: 20px 30px 0px 20px;
+}
+
+.button {
+  border: 1px solid black;
+  padding: 5px;
+  width: 30vw;
+  border-radius: 10px;
+  background-color: #b1c5c3;
+  margin: auto;
+}
+
+.tags {
+  text-decoration: none;
+  color: black;
+  font-size: 0.8rem;
+  padding: 25px 0;
+  text-align: left;
```

Comment:
you don't need this property

Reply:
I deleted color as I already have text-decoration. But when deleting other properties the tags look look displaced.